web/annotation_layer_builder.js

```diff
@@ -42,16 +41,18 @@ import { PresentationModeState } from "./ui_utils.js";
  *   [fieldObjectsPromise]
  * @property {Map<string, HTMLCanvasElement>} [annotationCanvasMap]
  * @property {TextAccessibilityManager} [accessibilityManager]
+ * @property {function} [onAppend]
```

Comment:
The same line is missing in the docs in the `web/annotation_editor_layer_builder.js` file.